frontends/common/parser_options.h

```diff
@@ -81,6 +81,8 @@ class ParserOptions : public Util::Options {
     std::filesystem::path file;
     /// if true preprocess only
     bool doNotCompile = false;
+    /// if true save preprocessed P4 to repro.p4
+    bool savePreprocessed = false;
```

Comment:
If you decide to address my first comment, then, obviously, this comment needs to be changed too.